Sample: JMXConnectionNotification_1
----------------------------------------------------------------
output:
./Data/javax_management_remote/javaFiles/JMXConnectionNotification_1.java:5: error: non-static variable this cannot be referenced from a static context
        JMXConnectionNotification notification = new JMXConnectionNotification("connectionOpened", this, 1, "Connection opened", "testConnectionId");
                                                                                                   ^
./Data/javax_management_remote/javaFiles/JMXConnectionNotification_1.java:5: error: constructor JMXConnectionNotification in class JMXConnectionNotification cannot be applied to given types;
        JMXConnectionNotification notification = new JMXConnectionNotification("connectionOpened", this, 1, "Connection opened", "testConnectionId");
                                                 ^
  required: String,Object,String,long,String,Object
  found:    String,JMXConnectionNotification_1,int,String,String
  reason: actual and formal argument lists differ in length
2 errors


================================================================

Sample: JMXConnectorFactory_2
----------------------------------------------------------------
stderr:
java.io.IOException: Failed to retrieve RMIServer stub: javax.naming.ServiceUnavailableException [Root exception is java.rmi.ConnectException: Connection refused to host: localhost; nested exception is: 
	java.net.ConnectException: Connection refused]
	at java.management.rmi/javax.management.remote.rmi.RMIConnector.connect(RMIConnector.java:370)
	at java.management/javax.management.remote.JMXConnectorFactory.connect(JMXConnectorFactory.java:270)
	at java.management/javax.management.remote.JMXConnectorFactory.connect(JMXConnectorFactory.java:229)
	at JMXConnectorFactory_2.main(JMXConnectorFactory_2.java:10)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at jdk.compiler/com.sun.tools.javac.launcher.Main.execute(Main.java:419)
	at jdk.compiler/com.sun.tools.javac.launcher.Main.run(Main.java:192)
	at jdk.compiler/com.sun.tools.javac.launcher.Main.main(Main.java:132)
Caused by: javax.naming.ServiceUnavailableException [Root exception is java.rmi.ConnectException: Connection refused to host: localhost; nested exception is: 
	java.net.ConnectException: Connection refused]
	at jdk.naming.rmi/com.sun.jndi.rmi.registry.RegistryContext.lookup(RegistryContext.java:138)
	at java.naming/com.sun.jndi.toolkit.url.GenericURLContext.lookup(GenericURLContext.java:220)
	at java.naming/javax.naming.InitialContext.lookup(InitialContext.java:409)
	at java.management.rmi/javax.management.remote.rmi.RMIConnector.findRMIServerJNDI(RMIConnector.java:1839)
	at java.management.rmi/javax.management.remote.rmi.RMIConnector.findRMIServer(RMIConnector.java:1813)
	at java.management.rmi/javax.management.remote.rmi.RMIConnector.connect(RMIConnector.java:302)
	... 10 more
Caused by: java.rmi.ConnectException: Connection refused to host: localhost; nested exception is: 
	java.net.ConnectException: Connection refused
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:626)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.newCall(UnicastRef.java:344)
	at java.rmi/sun.rmi.registry.RegistryImpl_Stub.lookup(RegistryImpl_Stub.java:116)
	at jdk.naming.rmi/com.sun.jndi.rmi.registry.RegistryContext.lookup(RegistryContext.java:134)
	... 15 more
Caused by: java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:593)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at java.rmi/sun.rmi.transport.tcp.TCPDirectSocketFactory.createSocket(TCPDirectSocketFactory.java:40)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	... 20 more

================================================================

Sample: JMXConnectorFactory_3
----------------------------------------------------------------
stderr:
java.io.IOException: Failed to retrieve RMIServer stub: javax.naming.ServiceUnavailableException [Root exception is java.rmi.ConnectException: Connection refused to host: localhost; nested exception is: 
	java.net.ConnectException: Connection refused]
	at java.management.rmi/javax.management.remote.rmi.RMIConnector.connect(RMIConnector.java:370)
	at java.management/javax.management.remote.JMXConnectorFactory.connect(JMXConnectorFactory.java:270)
	at JMXConnectorFactory_3.main(JMXConnectorFactory_3.java:14)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:569)
	at jdk.compiler/com.sun.tools.javac.launcher.Main.execute(Main.java:419)
	at jdk.compiler/com.sun.tools.javac.launcher.Main.run(Main.java:192)
	at jdk.compiler/com.sun.tools.javac.launcher.Main.main(Main.java:132)
Caused by: javax.naming.ServiceUnavailableException [Root exception is java.rmi.ConnectException: Connection refused to host: localhost; nested exception is: 
	java.net.ConnectException: Connection refused]
	at jdk.naming.rmi/com.sun.jndi.rmi.registry.RegistryContext.lookup(RegistryContext.java:138)
	at java.naming/com.sun.jndi.toolkit.url.GenericURLContext.lookup(GenericURLContext.java:220)
	at java.naming/javax.naming.InitialContext.lookup(InitialContext.java:409)
	at java.management.rmi/javax.management.remote.rmi.RMIConnector.findRMIServerJNDI(RMIConnector.java:1839)
	at java.management.rmi/javax.management.remote.rmi.RMIConnector.findRMIServer(RMIConnector.java:1813)
	at java.management.rmi/javax.management.remote.rmi.RMIConnector.connect(RMIConnector.java:302)
	... 9 more
Caused by: java.rmi.ConnectException: Connection refused to host: localhost; nested exception is: 
	java.net.ConnectException: Connection refused
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:626)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.createConnection(TCPChannel.java:217)
	at java.rmi/sun.rmi.transport.tcp.TCPChannel.newConnection(TCPChannel.java:204)
	at java.rmi/sun.rmi.server.UnicastRef.newCall(UnicastRef.java:344)
	at java.rmi/sun.rmi.registry.RegistryImpl_Stub.lookup(RegistryImpl_Stub.java:116)
	at jdk.naming.rmi/com.sun.jndi.rmi.registry.RegistryContext.lookup(RegistryContext.java:134)
	... 14 more
Caused by: java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:579)
	at java.base/sun.nio.ch.Net.connect(Net.java:568)
	at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:593)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
	at java.base/java.net.Socket.connect(Socket.java:633)
	at java.base/java.net.Socket.connect(Socket.java:583)
	at java.base/java.net.Socket.<init>(Socket.java:507)
	at java.base/java.net.Socket.<init>(Socket.java:287)
	at java.rmi/sun.rmi.transport.tcp.TCPDirectSocketFactory.createSocket(TCPDirectSocketFactory.java:40)
	at java.rmi/sun.rmi.transport.tcp.TCPEndpoint.newSocket(TCPEndpoint.java:620)
	... 19 more

================================================================

Sample: JMXConnectorFactory_4
----------------------------------------------------------------
stdout:
JMXConnector created successfully

================================================================

Sample: JMXConnectorServerFactory_3
----------------------------------------------------------------
output:
./Data/javax_management_remote/javaFiles/JMXConnectorServerFactory_3.java:9: error: unreported exception MalformedURLException; must be caught or declared to be thrown
        JMXServiceURL serviceURL = new JMXServiceURL("service:jmx:rmi:///jndi/rmi://localhost:9999/server");
                                   ^
1 error


================================================================

Sample: JMXPrincipal_1
----------------------------------------------------------------
stdout:
Principal name: admin

================================================================

Sample: JMXPrincipal_2
----------------------------------------------------------------
stdout:
true

================================================================

Sample: JMXPrincipal_3
----------------------------------------------------------------
stdout:
Hash code for JMXPrincipal: 92668751

================================================================

Sample: JMXPrincipal_4
----------------------------------------------------------------
stdout:
JMXPrincipal:  admin

================================================================

Sample: JMXServiceURL_1
----------------------------------------------------------------
output:
./Data/javax_management_remote/javaFiles/JMXServiceURL_1.java:5: error: unreported exception MalformedURLException; must be caught or declared to be thrown
        JMXServiceURL url = new JMXServiceURL("service:jmx:rmi:///jndi/rmi://localhost:9999/jmxrmi");
                            ^
1 error


================================================================

Sample: JMXServiceURL_2
----------------------------------------------------------------
output:
./Data/javax_management_remote/javaFiles/JMXServiceURL_2.java:7: error: unreported exception MalformedURLException; must be caught or declared to be thrown
        JMXServiceURL serviceURL = new JMXServiceURL("service:jmx:rmi:///jndi/rmi://localhost:9999/jmxrmi");
                                   ^
1 error


================================================================

Sample: JMXServiceURL_3
----------------------------------------------------------------
output:
./Data/javax_management_remote/javaFiles/JMXServiceURL_3.java:5: error: unreported exception MalformedURLException; must be caught or declared to be thrown
        JMXServiceURL url = new JMXServiceURL("service:jmx:rmi:///jndi/rmi://localhost:9999/jmxrmi");
                            ^
1 error


================================================================

Sample: JMXServiceURL_4
----------------------------------------------------------------
output:
./Data/javax_management_remote/javaFiles/JMXServiceURL_4.java:5: error: unreported exception MalformedURLException; must be caught or declared to be thrown
        JMXServiceURL url = new JMXServiceURL("service:jmx:rmi:///jndi/rmi://localhost:9999/server");
                            ^
1 error


================================================================

Sample: TargetedNotification_2
----------------------------------------------------------------
output:
./Data/javax_management_remote/javaFiles/TargetedNotification_2.java:6: error: incompatible types: String cannot be converted to Integer
        TargetedNotification targetedNotification = new TargetedNotification(new Notification("type", "source", 1), "listenerID");
                                                                                                                    ^
Note: Some messages have been simplified; recompile with -Xdiags:verbose to get full output
1 error


================================================================

